src/xbot/common/math/PID.java

```diff
@@ -55,22 +66,24 @@ public void reset()
      *  e.g. if you wanted a minimum rotation speed of 5 degrees per second,
      *  this tolerance would need to be 0.25.  
      */
-    public void setTolerances(double errorTolerance, double derivativeTolerance) {
+    public void setTolerances(double errorTolerance, double derivativeTolerance, double timeTolerenceInSeconds) {
```

Comment:
Yeah, the naming's a bit weird, since it's a tolerance about tolerances. I'll think about this some more.